scene/gui/control.h

```diff
@@ -265,8 +265,7 @@ class Control : public CanvasItem {
 public:
 	enum {
 
-		/*		NOTIFICATION_DRAW=30,
-		NOTIFICATION_VISIBILITY_CHANGED=38*/
+		// NOTIFICATION_VISIBILITY_CHANGED=38
```

Comment:
I guess `NOTIFICATION_DRAW` was here to indicate full CanvasItem notification range, to avoid overlaps if it's changed. In any case, it's completely unrelated to the issue, and probably should be left intact.

Reply:
Yea, I don't really remember why I removed that, ill add it back.